scout/parse/variant/variant.py

```diff
@@ -175,6 +176,12 @@ def parse_variant(variant, case, variant_type='clinical',
             dbsnp_ids.add(dbsnp)
         for cosmic in parsed_transcript.get('cosmic', []):
             cosmic_ids.add(cosmic)
+    
+    # The COSMIC tag in INFO is added via VEP and/or bcftools annotate
+
+    COSMIC_tag = variant.INFO.get('COSMIC')
```

Comment:
Please don't use capitals in variable names, it is not done anywhere else.
(I have to make some remark 😈 )